kairon/api/app/routers/bot/data.py

```
@@ -10,7 +11,7 @@
 from kairon.events.definitions.faq_importer import FaqDataImporterEvent
 from kairon.exceptions import AppException
 from kairon.shared.auth import Authentication
-from kairon.shared.cognition.data_objects import CognitionSchema
+from kairon.shared.cognition.data_objects import CognitionSchema, CognitionData
```

Ruff (0.8.2)

`kairon.shared.cognition.data_objects.CognitionData` imported but unused

Remove unused import: `kairon.shared.cognition.data_objects.CognitionData`

(F401)